htlcswitch/link.go

```diff
@@ -438,6 +449,7 @@ func (l *channelLink) Stop() {
 		l.cfg.ChainEvents.Cancel()
 	}
 
+	l.updateFeeTimer.Stop()
```

Comment:
We can't due to needing to reset the timer within`TestChannelLinkUpdateCommitFee`. Otherwise, we'll run into a race where we might update the timer without it being initialized first.